src/screens/SettingsScreen.js

```diff
@@ -80,12 +80,15 @@ export const SettingsScreen = () => {
       });
 
     const updateSectionedData = async () => {
-      const { settings = { matomo: false } } = globalSettings;
-      const pushPermission = await readFromStore(PushNotificationStorageKeys.IN_APP_PERMISSION);
+      const { settings = { matomo: false }, pushNotifications } = globalSettings;
```

Comment:
do we want to have `matomo` nested in `settings` but `pushNotifications` not? why should there be a difference?

Reply:
updated it to be in line with other "on/off" features